src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -281,6 +281,49 @@ function cones(PF::_FanLikeType)
   return IncidenceMatrix([Vector{Int}(x) for x in cones])
 end
 
+@doc raw"""
+    minimal_supercone(X::NormalToricVariety, p::AbstractVector{<:IntegerUnion})
```

Comment:
This is a method in polyhedral fans, so the input should be a polyhedral fan. Right?

If you allow for polyhedral fans as input, then they can have lineality. Right? Where is that case treated in your code?

Final question: Why not just return a cone? Why is an iterator needed/better here? At least from the name of the method, I expect a cone as output.

Reply:
If you really want a toric variety as first argument, then probably this method should be moved to the toric code.

Reply:
Moved to PolyhedralGeometry